index.html

```diff
@@ -19,7 +19,7 @@
         <!-- Navigation-->
         <nav class="navbar navbar-light bg-light static-top">
             <div class="container">
-                <a class="navbar-brand" href="#!">Start Bootstrap</a>

```

Comment:
starting a review attempt #2, please change the heading, font or color.

Reply:
we will work on that